### PR TITLE
Revised CTA Banner CSS changes - 579

### LINKED ISSUE #579: CTA Banner CSS changes
Irrespective of description, the when text and button is placed side by side, the gap is 32px. And when text and button are vertically stacked, gap is 24 px.

1. **With Description:**
  Till Mobile Screens the space between Description and CTA should be 24px. 
  From ipad to till large screens: It should be 32Px between Description and CTA .

2. **Without Description:** 
 Till Mobile Screens the space between Title and CTA should be 24px.

### PR DESCRIPTION
**Pull request by nagaraju-gspann**
Fix #579

Test URLs:
- Before: https://main--danaher-abcam--aemsites.hlx.page/en-us/page-test/cta-description
- After: https://579-CTA-CSS-Changes--danaher-abcam--aemsites.hlx.live/en-us/page-test/cta-description

### CODE REVIEW

File: eds/blocks/cta-banner/cta-banner.js [hunk 1]
```diff
@@ -6,25 +6,24 @@ import { applyClasses } from '../../scripts/scripts.js';
 export default async function decorate(block) {
   const productFullResponse = await getProductsListResponse(1);
   const parsedJson = JSON.parse(productFullResponse || '{}');
-  block.classList.add(...'cta-container flex md:items-center flex-col md:flex-row justify-between gap-8 md:gap-8 w-full p-6 mt-10 mb-14 rounded-xl bg-[#edf6f7]'.split(' '));
+  block.classList.add(...'cta-container flex md:items-center flex-col md:flex-row justify-between gap-6 md:gap-8 w-full p-6 mt-10 mb-14 rounded-xl bg-[#edf6f7]'.split(' '));
   const title = block.querySelector('h3');
-  applyClasses(title, 'title mt-0 mb-4 !leading-[22px] !font-bold text-xl text-[#378189] break-words');
```
Comment (shivangi1422) [inline eds/blocks/cta-banner/cta-banner.js:11]: make text-xl as important i.e !text-xl. It is taking the styling for h3 rather than text-xl

**Comment by nagaraju-gspann** [comment]
- Modified CSS changes according to revised Figma.
- Addressed & Fixed the review comments.
- Pushed and Raised PR.  

CC: @Meenu-GSPANN , @shivangi1422, @pardeepgera23

**Review by shivangi1422 — APPROVED** [review]

File: eds/blocks/cta-banner/cta-banner.js [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
+import { div, span } from '../../scripts/dom-builder.js';
 import { decorateIcons } from '../../scripts/aem.js';
 import { getProductsListResponse } from '../../scripts/search.js';
 import { applyClasses } from '../../scripts/scripts.js';
 
 export default async function decorate(block) {
   const productFullResponse = await getProductsListResponse(1);
   const parsedJson = JSON.parse(productFullResponse || '{}');
-  block.classList.add(...'cta-container flex md:items-center flex-col md:flex-row justify-between gap-8 md:gap-8 w-full p-6 mt-10 mb-14 rounded-xl bg-[#edf6f7]'.split(' '));
+  block.classList.add(...'cta-container flex md:items-center flex-col md:flex-row justify-between gap-6 md:gap-8 w-full p-6 mt-10 mb-14 rounded-xl bg-[#edf6f7]'.split(' '));
```
Comment (pardeepgera23) [inline eds/blocks/cta-banner/cta-banner.js:9]: Use applyClasses from scripts.js

**Review by pardeepgera23 — CHANGES_REQUESTED** [review]

File: eds/blocks/cta-banner/cta-banner.js (hunk 2 above)
Comment (nagaraju-gspann) [reply eds/blocks/cta-banner/cta-banner.js]: Fixed review comments.

**Review by pardeepgera23 — APPROVED** [review]

**Pull request merged**